Envfile

```diff
@@ -164,4 +164,21 @@ variable :SMARTY_STREETS_WEB_KEY, :String, default: "Optional"
 
 group :production do
   variable :SECRET_KEY_BASE, :String
+  
+  # Timber
+  variable :SEND_LOGS_TO_TIMBER, :String  # Defaults to true in code
+  variable :TIMBER, :String  # Only required if above is "true" or unset
```

Comment:
I know it's outside the scope here but it'd be great to have a link before each section that points to the service home page or something. Probably the fault here is to be found in the name of the variable, because `TIMBER` at first glance could be a token, a password, a URL or whatever. Only by grepping the code I found out it's a URL

Reply:
I went through and tried to add reasonable links to each service.  I'm not sure if you wanted to include that in this pull request or not, so I separated it out into its own commit. ☺️

Reply:
Damn, thats awesome! Thank you!

Reply:
thank you so much @rpalo 😭